src/widgets/DebouncedInput/DebouncedInput.vue

```diff
@@ -0,0 +1,38 @@
+<script lang="ts" setup>
+import { computed, onBeforeUnmount, ref } from 'vue';
+
+const props = defineProps({
+  modelValue: {
+    type: [String, Number],
+    required: true,
+  },
+  debounce: {
+    type: Number,
+    default: 250,
+  },
+});
+
+const emit = defineEmits(['update:modelValue']);
+
+const timeout = ref<ReturnType<typeof setTimeout>>();
+
+const localValue = computed({
+  get() {
+    return props.modelValue;
+  },
+  set(newValue) {
```

Comment:
Should we consider arrow functions for cases like this?
```suggestion
  get: () => props.modelValue,
  set: (newValue) => {
```